modules/olmv1-adding-a-catalog.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_content-type: PROCEDURE
+
+[id="olmv1-adding-a-catalog-to-a-cluster_{context}"]
+= Adding a catalog to a cluster
+
+To add a catalog to a cluster, create a catalog custom resource (CR) and apply it to the cluster.
+
+.Procedure
+
+. Create a catalog custom resource (CR), similar to the following example:
++
+.Example `redhat-operators.yaml`
+[source,yaml]
+----
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: redhat-operators
+spec:
+  source:
+    type: image
+    image:
+      ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <1>
+----
+<1> Specify the catalog's image in the `spec.source.image` field.
+
+. Run the following command to add the catalog to your cluster. By default, your catalog is installed in the `openshift-catalogd` namespace.
++
+[source,terminal]
+----
+$ oc apply -f redhat-operators.yaml
+----
++
+.Example output
+[source,text]
+----
+catalog.catalogd.operatorframework.io/redhat-operators created
+----
+
+.Verification
+
+* Run the following commands to verify the status of your catalog:
+
+.. Run the following command to check if your catalog is available:
++
+[source,terminal]
+----
+$ oc get catalog
+----
++
+.Example output
+[source,text]
+----
+NAME                  AGE
+redhat-operators      20s
+----
+
+.. Run the following command to check the status of your catalog pods:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-catalogd
+----
++
+.Example output
+[source,text]
+----
+NAME                                           READY   STATUS      RESTARTS   AGE
+catalogd-controller-manager-5b8844bbc4-ps76j   2/2     Running     0          60m
+redhat-operators                               0/1     Completed   0          47s
+----
```

Comment:
I would suggest changing this slightly such that it focuses only on the `Catalog` API.

The `Catalog` status has conditions, a resolved source, and a phase that are useful to determine the status of the catalog. Perhaps something that outputs `Catalog.status.phase` instead?

Reply:
@joelanford Do you know the command to output `Catalog.status.phase`?